### ISSUE TITLE
[FEATURE] Kuwait Theme - Category Buttons

### ISSUE THREAD

**Issue by FaithDaka** [IDEMSInternational/open-app-builder#2580]
**What?**
Creat and style  a button variant for category buttons

**Why?**
This variant is used to filter activities by category in the activity library

**How?**

- [ ] Add `category` variant to button variants
- [ ] Style variant
- [ ] Test variant with an example component

**Other details**
<img width="350" src="https://github.com/user-attachments/assets/2a07c26f-71e3-4c92-80ca-30fc74eec797">

**Comment by esmeetewinkel** [comment]
I interpreted these as radio buttons - could this be a variant of the `radio_button_grid` component? @jfmcquade 

`activity_library` [template](https://docs.google.com/spreadsheets/d/1V8iYNvVVc0whiZ3p--ulL_93DiOEH_SKwwwSqLX6OE4/edit?gid=0#gid=0) and [preview](https://plh-kids-kw-uat1.web.app/template/activity_library)
![Image](https://github.com/user-attachments/assets/403f00b0-c785-4144-b2e9-f0533475ad83)

**Comment by jfmcquade** [comment]
> I interpreted these as radio buttons - could this be a variant of the `radio_button_grid` component? (@jfmcquade please feed in)

Yes, I think that would make sense, I don't think it should be an issue to add them as a `variant` of `radio_button_grid` – perhaps named something like `circle` or `circle-icon`. 

@FaithDaka you can use a `@if (params.variant.includes("circle"))` block, like you did for the `task-progress-bar` component, to handle styling each radio item so that the text sits outside the container when that variant is applied
